src/context/LocaleContext.tsx

```diff
@@ -9,13 +9,7 @@ export const LocaleContext = createContext<LocaleContext>({
   currentLocale: locales['se-en'],
```

Comment:
Sorry this is out of context for this PR but does the `LocaleContext` need to be exported?

Reply:
Oh, not at all, well spotted! 🙏